srcpkgs/python3-pivy/template

```diff
@@ -13,18 +12,30 @@ license="0BSD"
 homepage="https://bitbucket.org/Coin3D/pivy"
 distfiles="https://bitbucket.org/Coin3D/pivy/get/${version}.tar.gz"
 checksum=2ee684dbf0f52c05b491c5423013fd82e679614bffee23641485248c8a790c4e
-nocross="require coin-config script"
 
 case "$XBPS_TARGET_MACHINE" in
 	ppc64*);;
 	ppc*) broken="{standard input}:3376761: Error: operand out of range";;
 esac
 
+if [ "$CROSS_BUILD" ]; then
+	CFLAGS+=" -I$XBPS_CROSS_BASE/usr/include/Coin3"
```

Comment:
while setting `CFLAGS` in template, appending shouldn't be used unless you set it to something before. It's harmless but I want you to point the exact behaviour.

So that `CFLAGS=...` should have the same effect.